Sources/StreamChatUI/Utils/ListChangeUpdater/ListChangeUpdater.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+/// Component responsible to process an array of `[ListChange<Item>]`'s and apply those changes to a view.
+protocol ListChangeUpdater {
```

Comment:
nit:
Maybe smth like `ListReloader` or `ListUpdater` would work better?

Reply:
I think that ListChangeUpdater is fine actually. `ListChange` prefix is related to `ListChange<Item>`. `Updater`, means it uses List Changes' to update the view 😄 Reloader is not really true because we can also delete items. `Reloader` feels like we just do `reloadData()`

Reply:
Maybe `ListChangesUpdater` could also work. But since for now, this is a private API, we can re-name and think about this later, WDTY?